examples/fig_setup.py

```diff
@@ -48,6 +48,14 @@ def get_gridspec():
         "membrane_schematic_1": gs_membranes[1, 0],
     }
 
+    if not independent_plots:
+        return gs_return_value
```

Comment:
Why not return this at the very top? No need for all these temporary objects to exist for no reason… 😉

Reply:
Initially I thought the original return value would be needed, but that is not the case. Changed it accordingly (the `gridspec`s have to exist though)